scarb/src/bin/scarb/args.rs

```diff
@@ -383,6 +383,10 @@ pub struct PackageArgs {
     #[arg(short, long)]
     pub list: bool,
 
+    /// Ignore warnings about a lack of human-usable metadata
+    #[arg(long)]
+    pub no_metadata: bool,
+
```

Comment:
Cargo actually doesn't have `no_metadata` argument in `cargo publish` (which I understand, because publishing a package with no information about it would be a bit misleading

Reply:
Yeah you are right, it's not an error, that's why I'm wondering now what is the use case for `--no-metadata` flag at all.

The thing is that all of these validations are only client-side and registry doesn't verify it at all. I guess its more of a friendly notice to the package creator to include info that might help others discover his package properly.

Also, the reason I've suggested it, is that I consider `package` args to be a subset of `publish` args - primarily because `package` is one of the steps of `publish` process. That's why for me, if there's an option to silence that kind of output in `package`, there should be an option to do that in `publish` too.

Reply:
Personally, it makes sense the way it is in Cargo (with the `--no-metadata` on package only). The package command is more generic - it only creates a tarball for distribution, but the "distribution" itself can be whatever. For some distribution methods it may make sense to omit the package metadata. On the other hand, it always makes sense to include metadata when publishing to registry.